### PR TITLE
Mod 9432  hover state change

### PR DESCRIPTION
**Pull request by chasls2**
**High level description:**

Add hover state to the video card thumbnail and title

**Technical details:**

added hover state to the video card thumbnail and title

**JIRA Ticket:**
[DEV-9432](https://federal-spending-transparency.atlassian.net/browse/DEV-9432)

**Mockup:**
https://invis.io/RYA3XN5WP#/273832670_Homepage_2-2_E

The following are ALL required for the PR to be merged:

Author:
- [x] Linked to this PR in JIRA ticket
- [x] Scheduled demo including Design/Testing/Front-end OR Provided instructions for testing in JIRA and PR `if applicable`
- [x] Verified cross-browser compatibility: Chrome, Safari, Firefox, Edge
- [x] Verified mobile/tablet/desktop/monitor responsiveness
- [x] Verified that this PR does not create any *new* accessibility issues (via Axe Chrome extension)
- [ ] Added Unit Tests for helper functions, reducers, models and Container/Component Interactivity Expectations `if applicable` [React Testing Library](react-testing-library.md)
- [ ] [API contract](https://github.com/fedspendingtransparency/usaspending-api/tree/dev/usaspending_api/api_contracts) updated `if applicable`
- [ ] [Component Library Integration Status Report](https://github.com/fedspendingtransparency/data-act-documentation/blob/data-transparency-ui/frontend_apps/component-library-integration-status.md) updated `if applicable`

Reviewer(s):
- [x] Design review complete `if applicable`
- [ ] [API #1234](https://github.com/fedspendingtransparency/usaspending-api/pull/1234) merged concurrently `if applicable`
- [ ] Code review complete

### CODE REVIEW

**Comment by chasls2** [comment]
ready for review

**Comment by anjenkin** [comment]
@chasls2 Make sure your PR is to merge to QAT not master.

File: src/_scss/pages/trainingVideos/_featuredVideo.scss [hunk 1]
```diff
@@ -62,6 +54,10 @@
       @media(min-width: $tablet-screen){
         font-size: $font-size-40;
       }
+      &:hover {
+        cursor: pointer;
+          text-decoration: underline;
+        }
     }
```
Comment (brianpetway) [inline src/_scss/pages/trainingVideos/_featuredVideo.scss:62]: at line 81 in this file you can remove the width: 100%
the width: 24 is the one that's being applied

File: src/_scss/pages/trainingVideos/_listOfVideos.scss [hunk 2]
```diff
@@ -31,6 +30,12 @@
...
     }
+    .video-card__headline{
+      &:hover{
+        text-decoration: underline;
+        cursor: pointer;
+      }
+    }
     .list-of-videos__inline{
       display:inline-flex;
       width: 100%;
```
Comment (brianpetway) [inline src/_scss/pages/trainingVideos/_listOfVideos.scss:41]: at line 46 in this file you can remove the width: 100%
the width: 24 is being applied

**Review by brianpetway — APPROVED** [review]

**Pull request merged**